validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -190,42 +195,38 @@ public void setUpLocalTests() {
      * create a bulk submission with one request, one customer, one individual requests and using the MCOL app
      */
     @Test
-    public void testAllSuccess() {
+    void testAllSuccess() {
 
         // set up a bulk customer to use the MCOL application
         bulkCustomer = createCustomer(createBulkCustomerApplications("MCOL"));
 
         // create an individual request
         final IIndividualRequest individualRequest = new IndividualRequest();
-        individualRequest.setId(requestId);
+        individualRequest.setId(REQUEST_ID);
         individualRequests = new ArrayList<>();
         individualRequests.add(individualRequest);
 
-        createBulkSubmission(numberOfRequests, bulkCustomer, individualRequests, "MCOL");
+        createBulkSubmission(NUMBER_OF_REQUESTS, bulkCustomer, individualRequests, "MCOL");
 
         // set up the mock objects
-        expect(mockIBulkCustomerDao.getBulkCustomerBySdtId(12345L)).andReturn(bulkCustomer);
-        replay(mockIBulkCustomerDao);
+        when(mockIBulkCustomerDao.getBulkCustomerBySdtId(12345L)).thenReturn(bulkCustomer);
 
         // inject the bulk customer into the validator
         validator.setBulkCustomerDao(mockIBulkCustomerDao);
 
-        // create a mock Bulk Submission DAO and return true
-        mockIBulkSubmissionDao = EasyMock.createMock(IBulkSubmissionDao.class);
-
-        expect(mockIBulkSubmissionDao.getBulkSubmission(bulkCustomer, bulkSubmission.getCustomerReference(), 90))
-                .andReturn(null);
-        replay(mockIBulkSubmissionDao);
+        when(mockIBulkSubmissionDao.getBulkSubmission(bulkCustomer, bulkSubmission.getCustomerReference(),
+                90)).thenReturn(null);
 
         validator.setBulkSubmissionDao(mockIBulkSubmissionDao);
 
         setupDataRetentionCache();
 
         bulkSubmission.accept(validator, null);
 
-        EasyMock.verify(mockIBulkCustomerDao);
-        EasyMock.verify(mockIBulkSubmissionDao);
-        EasyMock.verify(globalParameterCache);
+        verify(mockIBulkCustomerDao).getBulkCustomerBySdtId(12345L);
+        verify(mockIBulkSubmissionDao).getBulkSubmission(bulkCustomer, bulkSubmission.getCustomerReference(),
+                90);
+        //verify(globalParameterCache).getValue();
```

Comment:
Any reason this has been commented out?

Reply:
Reintroduced. And with parameters. *any* used deliberately as class is not accessible from this package.